src/components/calcite-tree-item/calcite-tree-item.tsx

```diff
@@ -149,6 +149,30 @@ export class CalciteTreeItem {
           <slot />
         </label>
       ) : null;
+    const bulletIcon =
+      this.selectionMode === TreeSelectionMode.Single ||
```

Comment:
Since `selectionMode` is referenced a few times, can you store it in a local var?

Reply:
I think this can be merged too:

```tsx
const icon = selectionMode === TreeSelectionMode.Children ? 
  "bullet-point" : selectionMode === TreeSelectionMode.Multi || selectionMode === treeSelectionMode.MultiChildren ? 
  "check" : null;
//...
const bulletOrCheckIcon = icon ?
<calcite-icon  
  class={{
    [CSS.bulletPointIcon]: icon === "bullet-point",
    [CSS.checkmarkIcon]: icon === "check",
    [CSS_UTILITY.rtl]: rtl
  }}
  icon={icon}
  scale="s"
/>
: null
```